experience.tex

```diff
@@ -6,33 +6,34 @@ \section{EXPERIENCE}
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Worked with the Supply Chain team to determine requirements and metric thresholds for displaying data to
-        the service providers based on their performance for previously completed work items.
+        Assisted building and maintaing greenfield iOS application written in SwiftUI to replace legacy Xamarin application,
+        reducing crash rate from 9\% to < 1\%, reducing time for local development and time to ship a new version of the app,
+        as well as added thorough logging to the application to make errors actionable for development team.
     }
     \item{
-        Utilized AWS Elasticsearch to improve filtering for assets by properties and to quickly return large lists of
-        matching data to the primary asset web service, lowering asset search response times for the end user.
+        Containerized legacy C\# applications to run in serverless AWS Fargate, moving secrets to HashiCorp Vault to be fetched
+        on container startup, and handled routing traffic securely from Route53, to ALB, to ECS cluster.
     }
     \item{
-        Assisted in translating UI mockups created by the development, sourcing, and UX teams to their final frontend versions using randomly generated placeholder data.
+        Triaged bug reports from users, frequently shipping a new mobile application within an hour of the error being reported.
```

Comment:
Do you have actual stats on this? Can you look back at hotfix branches that you've closed since? I know this is a nitpick but it seems like you're exaggerating from the language used, although I'm sure you're not in real life.

Reply:
Don't have the exact time a ticket was submitted to compare PRs to (it's all email based 🙃 ), but this is pretty frequent! Do you think changing this to: `within a few hours` or `same day` would make this seem more of a solid claim without actual stats?

The existing charter for the rebuild project has lines similar to:
```
Reduce the total amount of time it takes through running our build, testing, and deployment pipeline to submit a new version of the app.
Goal: > 1 hour
Current: 8 hours (Android/iOS)
```

But that isn't speaking to how fast issues can be triaged, fixed, and then deployed to the relevant app store. I think it's only speaking to the deployment part (unsurprising it would take that long given the legacy technical decisions made)

Reply:
🤷I think just say what you'll be most confident to support in an interview. I think the key here is the triage, deployment and validation to showcase your knowledge of software lifecycle

Reply:
Resolved in a call with you